### DISCUSSION TITLE
ImageMagick Installation Issue

### DISCUSSION THREAD

**Discussion by programmerarunkumar** [ImageMagick/ImageMagick#5419] [Help]
Getting the below error while installing the imagemagick in ubuntu 
[ErrorLog](https://docs.google.com/document/d/1LnKzdkxc_ooH9ynOq8qCiJivR9DO6nORiXltBP-aL30/edit?usp=sharing)

**Comment by urban-warrior** [comment]
Try `./configure --without-webp` and build again.    If you are C-savvy, take a look at the Webp headers.  `webp\mux.h` includes `webp\types.h` where WEBP_EXTERN is defined.  Why does it return an error, in particular, since we first call `webp\decode.h` which references WEBP_EXTERN as well and apparently did not cause any compile issues.  Perplexing indeed.

**Reply by programmerarunkumar** [reply]
I tried above command but now imagemagick is not supporting webp delegates
help me solve this issue 
I need to test with webp image format

**Reply by programmerarunkumar** [reply]
some one please help me on this 
I need to use IM for webp image format
